float64 shovel_angle